src/main/java/com/shippo/model/Pickup.java

```diff
@@ -0,0 +1,174 @@
+package com.shippo.model;
+
+import java.util.Map;
+
+import com.shippo.exception.APIConnectionException;
+import com.shippo.exception.APIException;
+import com.shippo.exception.AuthenticationException;
+import com.shippo.exception.InvalidRequestException;
+import com.shippo.net.APIResource;
+
+/**
+ * Use this class to schedule a pickup. Represents object defined in
+ * https://api.goshippo.com/pickups endpoint. The endpoint's documentation can be
+ * found in https://goshippo.com/docs/reference#pickups
+ */
+public class Pickup extends APIResource {
+	String objectId;
+	Object objectCreated;
+	Object objectUpdated;
+	Object carrierAccount;
+	Object location;
+	Object transactions;
+	Object requestedStartTime;
+	Object requestedEndTime;
+	Object confirmedStartTime;
+	Object confirmedEndTime;
+	Object cancelByTime;
+	Object status;
+	Object confirmationCode;
+    Object timezone;
```

Comment:
spacing weirdness here as well

Reply:
Spaces & Tabs used in different files, I've installed a linter to fix this